internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1056,6 +1058,8 @@ func (r *AgentServiceConfigReconciler) newImageServiceStatefulSet(ctx context.Co
 			{Name: "HTTPS_CA_FILE", Value: "/etc/image-service/ca-bundle/service-ca.crt"},
 			{Name: "ASSISTED_SERVICE_SCHEME", Value: "https"},
 			{Name: "ASSISTED_SERVICE_HOST", Value: serviceName + "." + r.Namespace + ".svc:" + servicePort.String()},
+			{Name: "IMAGE_SERVICE_SCHEME", Value: imageServiceScheme},
+			{Name: "IMAGE_SERVICE_HOST", Value: imageServiceHost},
```

Comment:
Can you add a test to ensure these are set correctly?

Reply:
sure, added